src/integrations.html

```diff
@@ -0,0 +1,316 @@
+{{#> layout page="integrations"}}
+
+<section id="intro">
+  <div class="m-auto max-w-7xl px-4 pt-10">
+    <h2 class="text-gray-900 text-3xl font-normal text-left">
+      Integrations
+    </h2>
+    <div class="mt-3 lg:w-[520px]">
+      <span class=" text-gray-700 font-normal text-left">
+        Livebook comes with built-in integrations with Elixir, multiple data sources, data visualization libraries, and more!
+      </span>
+    </div>
+  </div>
+</section>
+
+<section id="integrations">
+  <div class="m-auto max-w-7xl px-4 pt-10 lg:mb-20 flex flex-col lg:flex-row lg:items-start gap-5 lg:gap-16">
+    <div id="btn-integration" class="hidden lg:flex flex-col gap-1">
+      <button data-type="all" class=" button-integration button-integration-active">All</button>
+      <button data-type="database" class="button-integration">Database & <br> Data Warehouse</button>
+      <button data-type="language" class="button-integration">Language</button>
+      <button data-type="machine-learning" class="button-integration">Machine Learning</button>
+      <button data-type="messaging" class="button-integration">Messaging</button>
+      <button data-type="visualization" class="button-integration">Visualization</button>
+      <div class="hidden lg:block w-60 mt-6 p-5 bg-blue-50 border rounded-lg border-blue-600">
+        <span class="font-semibold">Can't find an integration?</span>
+        <div class="mt-2 text-sm">
+          <span>
+            Integrations are hosted as open-source packages
+            <a
+              class="underline"
+              href="https://hex.pm/packages?search=kino&sort=recent_downloads"
+              target="_blank"
+            >on Hex.pm</a>.
+            Anyone can build and publish new integrations. Use our
+            <a
+              class="underline"
+              href="https://github.com/livebook-dev/livebook/discussions/categories/integrations"
+              target="_blank"
+            >GitHub Discussions</a>
+            to suggest new integrations and discuss with the community.
+          </span>
+        </div>
+      </div>
+    </div>
+    <select onchange="filterSelection(value)" class="lg:hidden block w-full p-2.5 bg-gray-200 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-gray-500 focus:border-gray-500">
```

Comment:
We need to addEventListener for the select change, and call `filterSelection` from there :)